staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/api.proto

```diff
@@ -77,3 +79,31 @@ message TopologyInfo {
 message NUMANode {
     int64 ID = 1;
 }
+
+// DynamicResource contains information about the devices assigned to a container by DRA
+message DynamicResource {
+    string class_name = 1;
+    string claim_name = 2;
+    string claim_namespace = 3;
+    repeated CDIDevice cdi_devices = 4;
```

Comment:
Let's wrap `CDIDevice` in another message in order to future proof it in case we decide to add the `KubeletPluginName` later